eva-lib/src/test/java/uk/ac/ebi/eva/lib/metadata/MetadataTestConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package uk.ac.ebi.eva.lib.metadata;
+
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+import uk.ac.ebi.eva.lib.extension.ExtendedJpaRepositoryFunctionsImpl;
+
+@SpringBootConfiguration
+@ComponentScan
+@EntityScan(basePackages = {"uk.ac.ebi.eva.lib.entity"})
+@EnableJpaRepositories(basePackages = {"uk.ac.ebi.eva.lib.repository"}, repositoryBaseClass = ExtendedJpaRepositoryFunctionsImpl.class)
+public class MetadataTestConfiguration {
```

Comment:
This class is used by Spring to initialise the repositories and entities used in the test (it will create a in memory database containing all the entities). 

Like this module is a library, there is no spring boot application annotation (that would do the same) and we need this class (removing it will make the tests not work).